src/komodo_validation.h

```diff
@@ -76,14 +76,15 @@ extern char ASSETCHAINS_SYMBOL[65]; // defined in validation.cpp
 // 2: ENDS: July 15th 2019 1563148800 -> estimated height 4,173,578
 // 3: ENDS: Sunday, June 14th, 2020 03:00:00 PM UTC
 // 4: ENDS: Monday, June 14th, 2021 (03:00:00 PM UTC)
-#define NUM_KMD_SEASONS 5
+// 5: ENDS: Friday, June 24th, 2022 (01:37:33 PM GMT)
+#define NUM_KMD_SEASONS 6
 #define NUM_KMD_NOTARIES 64
 // first season had no third party coins, so it ends at block 0. 
 // second season ends at approx block 4,173,578, please check this!!!!! it should be as close as possible to July 15th 0:00 UTC. 
 // third season ended at AYA height 335000
 // Fourth season ends at around June 14th, 2021 (03:00:00 PM UTC) ,AYA block number: 770700
-// Fifth season end is unknown so is set to 999999999
-static const int32_t KMD_SEASON_HEIGHTS[NUM_KMD_SEASONS] = {0, 335000, 398750, 770700, 999999999};
+// Fifth season end at around June 24th, 2022 (13:37:33 GMT) ,AYA block number: 1128667
```

Comment:
current block [1102331](https://ayablockexplorer.com/block/1102331)
HF block: 1128667
diff = 26336
If going back 26336 blocks from present (block [1075994](https://ayablockexplorer.com/block/dd180ed11910f95095f290e201679a3065b4ebac0059646f7fb25d2a5bba6dc5) that is 13th May (i.e. around a month). If target date is June 24th, this block height wont appear for a couple of weeks later.
Please recheck the maths here, using recent actual block production rate to get a closer estimated block height.

Reply:
Block height updated to 1110400